pkg/reaper/nodereaper/helpers.go

```diff
@@ -209,6 +227,123 @@ func (ctx *ReaperContext) publishEvent(namespace string, event *v1.Event) error
 	return nil
 }
 
+func (ctx *ReaperContext) obtainReapLock(ddbAPI dynamodbiface.DynamoDBAPI, nodeName, instanceID, nodeType string) (LockRecord, error) {
+	log.Infof("obtaining lock for a %s node %s (%s)", nodeType, nodeName, instanceID)
+
+	timestamp := time.Now().Format(time.RFC3339)
+
+	lock := LockRecord{
+		LockType:   nodeType,
+		ClusterID:  ctx.ClusterID,
+		NodeName:   nodeName,
+		InstanceID: instanceID,
```

Comment:
Aren't these duplicative of each other? Will we ever have collisions on `nodeName` or `instanceid`? Or is this purely for debuggability?

Reply:
Yea, this is just to make things easier should someone look at the lock.